x/btcstaking/client/cli/tx.go

```diff
@@ -215,67 +214,72 @@ func NewCreateBTCDelegationCmd() *cobra.Command {
 				return err
 			}
 
-			// get staking tx info
-			stakingTxInfo, err := btcctypes.NewTransactionInfoFromHex(args[2])
+			// get staking tx bytes
+			stakingTx, err := hex.DecodeString(args[2])
+			if err != nil {
+				return err
+			}
+
+			inclusionProof, err := types.NewInclusionProofFromHex(args[3])
```

Comment:
shouldn't we call the `types.NewInclusionProofFromHex(` only if `len(args[3]) > 0`  i.e if user provides empty string as inclusion proof we create message where inclusion proof is `nil` ?